### ISSUE TITLE
Difficulty reading South African Licence Disk

### ISSUE THREAD

**Issue by NardusRetroRabbit** [googlesamples/android-vision#343]

**Comment by Jody-Geers** [comment]
Been fighting with this for a few days - I think it might be the quality of the code printed on the cards. ( even hd scanned card, cropped barcode, adjusted bright / contrast, and load from file ) - Still no luck.

**Comment by NardusRetroRabbit** [comment]
@Jody-Geers , yeah I ended up using another service as I could not get it to work.

**Comment by Jody-Geers** [comment]
@NardusRetroRabbit Android SDK? Free service? Please Share, I'm pulling my hair out here . . .

**Comment by NardusRetroRabbit** [comment]
@Jody-Geers , it is an SDK, but unfortunately it is quite expensive. Manatee Works, but it works flawlessly.

**Comment by Jody-Geers** [comment]
@NardusRetroRabbit Thank Bru! (B) - Only problem, my previous comment about the print quality is now dismissed if another service can read it. Which means, you'r original point is more valid than ever. . . . Which is a real pity, i had massive hopes for this google stuffs. ( sucks there is no other free alternative )

**Comment by Jody-Geers** [comment]
@NardusRetroRabbit sharing is caring ... there is an encryption? 
https://mybroadband.co.za/vb/showthread.php/382187-Decode-Drivers-Licence-Barcode
https://pastebin.com/gb049dfx
